src/Pickles/Pickles.DocumentationBuilders.Json/JsonTableRow.cs

```diff
@@ -18,22 +18,26 @@
 //  </copyright>
 //  --------------------------------------------------------------------------------------------------------------------
 
-using System;
 using System.Collections.Generic;
 
 namespace PicklesDoc.Pickles.DocumentationBuilders.Json
 {
-    public class JsonTableRow : List<string>
+    public class JsonTableRow : List<object>
     {
         public JsonTableRow()
         {
         }
 
-        public JsonTableRow(params string[] cells)
+        public JsonTableRow(IEnumerable<string> cells)
         {
             AddRange(cells);
         }
 
+        public void AddResultToJsonTableRow()
```

Comment:
Since the only place where this method is called, is right next to a call to the constructor, I suggest to add the result to the parameter list of the constructor, set the result in the constructor, add the result to the collection in the constructor, remove the AddResultToJsonTableRow method and make the Result property readonly.

That makes it more explicit what's going on, and reduces the danger of adding the result twice.